### PR TITLE
Update README for ui development.

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes

Updates README for building and making changes to ui codebase.

## Related issue number (if any)
https://linear.app/aqueducthq/issue/ENG-1280/cleanup-outdated-ui-documentation-in-our-repo
https://linear.app/aqueducthq/issue/ENG-1268/update-ui-readme-file

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have manually run the integration tests and they are passing.
- [x] All features on the UI continue to work correctly.

### CODE REVIEW

File: src/ui/README.md [hunk 1]
```diff
@@ -4,14 +4,61 @@ The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
 package, you can start the UI by running `aqueduct ui`.
```
Comment (cw75) [inline src/ui/README.md:4]: This `aqueduct ui` is also deprecated. Could you update this to "The UI is started together with the backend server via `aqueduct start`"?

File: src/ui/README.md [hunk 2]
```diff
@@ -4,14 +4,61 @@ The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
...
 
 The code here is organized into two directories -- a component library in
-`components/` that is published to npm, and an application server in `app/`
-that consumes `components/`.
+`common/` that is published to npm as a package called `@aqueducthq/common`, and an application server in `app/`
+that consumes `@aqueducthq/common`.
 
 If you're actively developing the UI, you will need to copy the `.env.local` file
```
Comment (cw75) [inline src/ui/README.md:10]: @vsreekanti @agiron123  is the `.env.local` file still relevant? I don't think we have `~/.aqueduct/ui/app` folde anymore right? The directory just contains a few "compiled" UI files.

**Review by cw75 — COMMENTED** [review]
LGTM for the second part. Left a couple comments in the first part.

File: src/ui/README.md (hunk 1 above)
Comment (agiron123) [reply src/ui/README.md:4]: sure can

File: src/ui/README.md (hunk 2 above)
Comment (agiron123) [reply src/ui/README.md:10]: @vsreekanti @cw75 just want to confirm that all we need in env.local is the SERVER_ADDRESS field, correct?
```
SERVER_ADDRESS=http://localhost:8080
```

**Comment by vsreekanti** [comment]
Yep, that seems right: https://github.com/aqueducthq/aqueduct/blob/410ed5b1684c0dd95154176495cda53c0684a13a/src/ui/common/src/components/hooks/useAqueductConsts.ts#L5-L9

File: src/ui/README.md [hunk 3]
```diff
@@ -1,17 +1,69 @@
...
+This file should contain the following:
+
+```SERVER_ADDRESS=http://localhost:8080```
+
+If you are exposing the ui at a public IP, make
 sure to update the `SERVER_ADDRESS` field accordingly (e.g. 
-`SERVER_ADDRESS=3.142.105.48:8080`). Then, you can start the Next.js server in dev 
-mode by running `make dev`. Note that this will symlink your local version of
-the `components/` directory to pick up any changes you might have made. 
+`SERVER_ADDRESS=3.142.105.48:8080`).
```
Comment (vsreekanti) [inline src/ui/README.md:18]: Stray close parens.

Comment (vsreekanti) [reply src/ui/README.md:18]: This isn't about exposing the UI at a public IP. This is about whether the serving is running on a different machine than the UI.

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**